### PR TITLE
CSS Background: background-position.md file created

### LINKED ISSUE #3537: [Edit] CSS [Edit] CSS Background: background-position
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like an edit a term within the CSS Animations entry. The entry lives in docs/content/css/concepts/animations/terms/background-position/background-position.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by luv2027**
Fixes: #3537

### Description

This PR added a file named background-position.md file in docs/content/css/concepts/animations/terms/background-position/background-position.md directory.

### Type of Change

- Adding a new file.

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #3537 in the "Description" above).

### CODE REVIEW

File: content/css/concepts/animations/terms/background-position/background-position.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+Title: 'background-position'
```
Comment (caupolicandiaz) [inline content/css/concepts/animations/terms/background-position/background-position.md:2]: This is incorrect. This file already exists - https://www.codecademy.com/resources/docs/css/background/background-position

The PR is for the addition of images that match the examples in the existing entry. For example,
https://www.codecademy.com/resources/docs/css/animations/animation

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@luv2027 Please review my comments carefully and update the PR accordingly.

File: content/css/concepts/animations/terms/background-position/background-position.md (hunk 1 above)
Comment (luv2027) [reply content/css/concepts/animations/terms/background-position/background-position.md:2]: can you tell me in which file the images needs to be added? The issue raised said that this file did not existed so I created one.

Comment (caupolicandiaz) [reply content/css/concepts/animations/terms/background-position/background-position.md:2]: There should be images added that represent the examples in the entry

**Comment by caupolicandiaz** [comment]
@luv2027 I'm closing the PR due to lack of progress

**Pull request closed without merging**